docs/appendices/release-notes/5.7.0.rst

```diff
@@ -75,13 +75,14 @@ SQL Standard and PostgreSQL Compatibility
 - Added initial support for foreign data wrappers, including a ``jdbc``
   :ref:`foreign data wrapper <administration-fdw>`.
 
-- Added support for lower case ``yyyy`` format string to
-  :ref:`to_char <scalar-to_char>` scalar function, which behaves exactly the
-  same as uppercase ``YYYY``, to match PostgreSQL behaviour.
-
 - Added the :ref:`scalar-current_role` scalar function as an alias for
   :ref:`scalar-current_user`.
 
+- `Dhruv Patel <https://github.com/DHRUV6029>`_ added support for the ``^``
+  operator as syntax sugar for the :ref:`scalar-power` scalar function. E.g.::
+
+    SELECT 5 ^ 2;
```

Comment:
Went to compability section as we had already such scalar, alias for compatibility (same as current_role above)